app/views/allocations/history.html.erb

```diff
@@ -36,6 +36,10 @@
                                       allocation_type: 'Co-working',
                                       pom_name: allocations[i + 1].secondary_pom_name,
                                       pom_email: @pom_emails[allocations[i + 1].secondary_pom_nomis_id] } %>
+            <% elsif allocation.event == 'deallocate_released_offender' %>
+              <%= render :partial => "deallocate_released_offender",
```

Comment:
would be good to rebase on mo-56-alloc-history-partial-path - then this if statement goes away